tests/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# This is the cmake build file for the tests directory of the
+# NCEPLIBS-g2c_UTILS project.
+#
+# Ed Hartnett, 7/11/21
+
+if(CMAKE_Fortran_COMPILER_ID MATCHES "^(Intel)$")
+  set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -r8 -convert big_endian -assume byterecl")
+elseif(CMAKE_Fortran_COMPILER_ID MATCHES "^(GNU)$")
+  set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -ffree-line-length-0 -fdefault-real-8 -fconvert=big-endian")
+endif()
```

Comment:
Why is this added? There's no Fortran in this repository. Should be removed.

Reply:
OK, will do. Good catch.